include/setup.h

```diff
@@ -250,16 +250,22 @@ class Prop_string:public Property{
 	virtual bool CheckValue(Value const& in, bool warn);
 	~Prop_string(){ }
 };
-class Prop_path:public Prop_string{
+
+class Prop_path : public Prop_string {
 public:
-	std::string realpath;
-	Prop_path(std::string const& _propname, Changeable::Value when, char const * const _value)
-		:Prop_string(_propname,when,_value) {
-		default_value = value = _value;
-		realpath = _value;
+	Prop_path(const std::string &name, Changeable::Value when, const char *val)
+	        : Prop_string(name, when, val),
+	          realpath(val)
+	{
+		default_value = val;
+		value = val;
```

Comment:
Can: `default_value = val;` and `value = val;` be moved into the initialization list too?

Reply:
They can't be moved to initialization list, because they are fields in base class… But thanks for pointing this out - these fields should *not* be initialized here at all! They are already initialized via `Prop_string` constructor.